site2/tools/publish-website.sh

```diff
@@ -52,11 +51,7 @@ PULSAR_SITE_TMP=/tmp/pulsar-site
     mkdir -p $PULSAR_SITE_TMP/content/
   fi
 
-  echo "BUILD_ALL:"$BUILD_ALL
-  if [[ $BUILD_ALL"" == "11" ]]; then
-    echo "clean all the old random js files because a new all version build done, all the js is regenerated with a random name"
-    rm -rf $PULSAR_SITE_TMP/content/assets/js
-  fi
+  rm -rf $PULSAR_SITE_TMP/content/*
```

Comment:
you can't remove all content every time, just when the website has a full build, you can do that, in fact, we just build the versions which have files changes, just once full build(all versions and all languages) one day, so these code can't be removed

Reply:
It seems a full build takes 40 minutes to complete, while we normally build every six hours. Do we really need this improvement?

The current issue is that if we don't remove content, we can have dangling pages even if it's removed on the master branch.

But yes, we can retain the current logic while removing all content when build all :)

Reply:
Updated.